node/hyperbahn/handler.js

```diff
@@ -24,10 +24,14 @@
 
 var assert = require('assert');
 var util = require('util');
+var fs = require('fs');
+var path = require('path');
 var setTimeout = require('timers').setTimeout;
 
 var Errors = require('../errors.js');
 var TChannelJSON = require('../as/json');
+var TChannelThrift = require('../as/thrift');
```

Comment:
thriftify/thriftrw is just a matter of how the objects get serialized on the wire, no? Can we not swap this out with thriftrw later?